app/models/piece.rb

```diff
@@ -0,0 +1,7 @@
+class Piece < ActiveRecord::Base
```

Comment:
Sounds good. We could always add this as part of the STI ticket, so there's still a good opportunity, but I'm also fine making do without.